godot_project/editor/controls/workspace_view/tab_bar_active_workspaces.gd

```diff
@@ -40,6 +40,22 @@ func _ready() -> void:
 	MolecularEditorContext.homepage_activated.connect(_on_homepage_activated)
 
 
+func _process(delta: float) -> void:
+	var tab_bar_rect: Rect2 = tab_bar_active_workspaces.get_global_rect()
```

Comment:
do we need to perform this in process? Is there any 'mouse event' signal that we could use instead?

Reply:
yes